osu.Game/Rulesets/Mods/ModMirror.cs

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+namespace osu.Game.Rulesets.Mods
+{
+    public abstract class ModMirror : Mod
+    {
+        public override string Name => "Mirror";
+        public override string Acronym => "MR";
+        public override ModType Type => ModType.Conversion;
+        public override string Description => "Flips the beatmap horizontally.";
```

Comment:
```suggestion
        public override string Description => "Flips the playfield horizontally.";
```

Also I'm thinking of adding a setting to flip it vertically (if in scope of this PR).